about.html

```diff
@@ -0,0 +1,108 @@
+        </ul>
+      </div>
+      <div class="nav-together container-desk">
+        <img src="assets/icons/x-icon.svg" class="X-icon" alt="closing menu">
+        <a href="index.html"><img src="assets/images/act/Logo.png" class="logo-image" alt="logo image"></a>
+        <nav class="nav-bar" id="nav-bar">
+          <ul class="nav-list">
+            <li><a href="index.html" class="nav-item">Home</a></li>
+            <li><a href="index.html" class="nav-item">Program</a></li>
+            <li><a href="index.html" class="nav-item">Join</a></li>
+            <li><a href="index.html" class="nav-item">Sponsor</a></li>
+            <li><a href="index.html" class="nav-item">News</a></li>
+          </ul>
+        </nav>
+        <button class="nav-top-button"><a href="about.html">Join the fight!</a></button>
+        </div>
+        <img src="assets/icons/ham-icon.svg" class="ham-icon" alt="menu">
+    </header>
+    <main class="container-fluid container-default aboutpage">
+      <section class="fade-image">
+        <span class="img"></span>
+    </section>
+    <section class="rest-aboutpage">
+      <section class="about-text-section">
+        <h2 class="about-text-2">About Us</h2>
+        <article class="outline">
+        <p class="about-paragraph-home-1">  We are a grobal company that fights for the rights for those who cannot fight for themselves.<br><br>We would love to here from you and what we can do for you.Advocating for positive change with people over 80 countries and growing!</p>
+        </article>
+        <p class="p2">Please contact us or email us at this address if you want any form of help or questions about our mission and countires we operate from.<br><span> justiceforall@gmail.com</span></p>
+      </section>
+      <section class="mission">
+        <article class="blackheader">
+          <h2 class="main-text">Our Mission</h2>
+          <img src="assets/images/patterns/underline.png" alt="underline for text">
+        </article>
+        <p class="missionstatement">Our mission is to give every one a voice in the world and have every right to resources,education,gorvenment and civilisation among other things</p>
+        <img src="assets/images/mission.png" class="imagmission" alt="people s arms image">
+      </section>
+      <section class="past">
+        <article class="blackheader">
+          <h2 class="main-text">Past Trials</h2>
+          <img src="assets/images/patterns/underline.png" alt="underline for text">
+        </article>
+        <p class="missionstatement">take a look at some of our past trials and how we handled them with leading experts in law.</p>
+
+        <section class="past-images">
+          <figure class="img1"><img src="assets/images/haiti.jpg"  class="img-deep" alt="haiti images"></figure>
+          <figure class="img"><img src="assets/images/ukraine.jpg" class="img-deep" alt="haiti images"></figure>
```

Comment:
- [optional] you can reduce the image sizes. they kind of look too big compared to the main design.